### PR TITLE
Feature/add fixtures

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/FOR-238

- Add some fixtures to the site

### CODE REVIEW

File: web/modules/custom/foreningsmentor_fixtures/.DS_Store
Comment (tuj) [inline web/modules/custom/foreningsmentor_fixtures/.DS_Store:1]: Remove

File: web/modules/custom/foreningsmentor_fixtures/assets/.DS_Store
Comment (tuj) [inline web/modules/custom/foreningsmentor_fixtures/assets/.DS_Store:1]: remove

**Review by tuj — APPROVED** [review]

File: web/modules/custom/foreningsmentor_fixtures/.DS_Store
Comment (martinyde) [reply web/modules/custom/foreningsmentor_fixtures/.DS_Store:1]: Woopsie

File: web/modules/custom/foreningsmentor_fixtures/assets/.DS_Store
Comment (martinyde) [reply web/modules/custom/foreningsmentor_fixtures/assets/.DS_Store:1]: woopsie

**Pull request merged**